vapi/simulator/simulator.go

```diff
@@ -771,10 +779,16 @@ func (s *handler) library(w http.ResponseWriter, r *http.Request) {
 			}
 			OK(w, ids)
 		case "":
+			if !s.isValidSecurityPolicy(spec.Library.SecurityPolicyID) {
+				BadRequest(w, "com.vmware.vapi.std.errors.invalid_argument")
```

Comment:
I had started working on this a while back and iirc this case would be `http.NotFound(w, r)` ?

Reply:
Yeah, I think you're right. I'll update.